Makefile

```diff
@@ -6,4 +6,27 @@ docs: contrib/docs/rebuilderd.1 contrib/docs/rebuildctl.1 contrib/docs/rebuilder
 contrib/docs/%: contrib/docs/%.scd
 	scdoc < $^ > $@
 
-.PHONY: all docs
+install:
```

Comment:
`install: all docs` would make it sure that things are built before trying to be installed.